mitm_proxy/src/requests.rs

```diff
@@ -109,7 +112,20 @@ impl RequestInfo {
         });
 
         row.col(|ui| {
-            ui.label(req.method().to_string());
+            let method = req.method();
+            let color = match *method {
+                Method::OPTIONS => Color32::GRAY,
+                Method::GET => Color32::GREEN,
+                Method::POST => Color32::BLUE,
+                Method::PUT => Color32::LIGHT_BLUE,
+                Method::DELETE => Color32::RED,
+                Method::HEAD => Color32::YELLOW,
+                Method::TRACE => Color32::BROWN,
+                Method::CONNECT => Color32::GOLD,
+                Method::PATCH => Color32::DARK_BLUE,
+                _ => Color32::DARK_GRAY,
+            };
```

Comment:
probably a better solution should to use a separate function to manage colors
`let color = get_color(method)`